exe-unit/resources/local-debug-exeunits-descriptor.json

```diff
@@ -8,5 +8,14 @@
         "properties": {
             "wasm.wasi.version@v": "0.9.0"
         }
+    },
+    {
+        "name": "vm",
+        "version": "0.1.0",
+        "supervisor-path": "../../target/debug/exe-unit",
+        "runtime-path": "../../../ya-runtime-vm/target/debug/ya-runtime-vm",
```

Comment:
Will try to update `ya-runtime-vm` so it will create `target/debug/runtime` dir during debug build, and then will merge this PR.